feature/home/src/main/java/com/epmedu/animeal/model/FeedingPoint.kt

```diff
@@ -0,0 +1,52 @@
+package com.epmedu.animeal.model
+
+import com.epmedu.animeal.resources.R
+import com.mapbox.geojson.Point
+
+data class FeedingPoint(
+    val id: Int, // For future implementations
+    val priority: Int,
+    val status: State,
+    val animalType: AnimalType,
+    val isFavourite: Boolean = false,
+    val remoteness: Remoteness = Remoteness.ANY,
+    var coordinates: Point
+) {
+
+    fun getDrawableRes(): Int =
+        when {
+            isFavourite -> {
+                when (status) {
+                    State.RED -> R.drawable.ic_favstate_favouritehungry_high
+                    State.YELLOW -> R.drawable.ic_favstate_favouritehungry_medium
+                    State.GREEN -> R.drawable.ic_favstate_favouritehungry_low
+                }
+            }
+            animalType == AnimalType.DOG -> {
+                when (status) {
+                    State.RED -> R.drawable.ic_dogsstate_doghungry_high
+                    State.YELLOW -> R.drawable.ic_dogsstate_doghungry_medium
+                    State.GREEN -> R.drawable.ic_dogsstate_doghungry_low
+                }
+            }
+            else -> {
+                when (status) {
+                    State.RED -> R.drawable.ic_catsstate_cathungry_high
+                    State.YELLOW -> R.drawable.ic_catsstate_cathungry_medium
+                    State.GREEN -> R.drawable.ic_catsstate_cathungry_low
+                }
+            }
+        }
+}
+
+enum class State {
+    RED, GREEN, YELLOW
+}
+
+enum class Remoteness {
+    NEAREST, ANY, NEAREST_10_KM
+}
+
+enum class AnimalType {
```

Comment:
You don't need to create this enum since we already have Tab enum in AnimealSwitch, just rename it and use it in this data class

Reply:
I'll do.